docs/iris/src/whatsnew/latest.rst

```diff
@@ -77,12 +103,35 @@ Incompatible Changes
   :mod:`iris.util` module.  Please use the :func:`iris.util.equalise_attributes`
   function instead.
 
-* The :mod:`iris.experimental.concatenate` module has now been removed. In 
+* The :mod:`iris.experimental.concatenate` module has now been removed. In
   ``v1.6.0`` the experimental ``concatenate`` functionality was moved to the
   :meth:`iris.cube.CubeList.concatenate` method.  Since then, calling the
-  :func:`iris.experimental.concatenate.concatenate` function raised an 
+  :func:`iris.experimental.concatenate.concatenate` function raised an
   exception.
 
+Internal
+========
+
+* Changed the numerical values in tests involving the Robinson projection due
+  to improvements made in `Proj <https://github.com/OSGeo/PROJ>`_.
+
+* Change tests to account for changes in behaviour in
+  `GDAL <https://github.com/OSGeo/gdal>`_.
```

Comment:
I'd like to see some minor detail on what the changes are that we've responded to - similar to the 3 other `Internal` bullets below.

Reply:
I've split this into two bullet points going into more detail.

Reply:
I like it, but see https://github.com/SciTools/iris/pull/3811#pullrequestreview-482628303 for more.